recipes/proselint/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "proselint" %}
+{% set version = "0.10.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "3a87eb393056d1bc77d898e4bcf8998f50e9ad84f7b9ff7cf2720509ac8ef904"
+
+build:
+  number: 0
+  entry_points:
+    - proselint = proselint.command_line:proselint
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - click
```

Comment:
These are listed in "install_requires" so they seem like real host deps

Reply:
you only need `setup_requires`